lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
         }
 
         /// <summary>
+        /// Sets Per Map Configuration
```

Comment:
I'm thinking the IMRUDriver could have a constructor parameter of type `Set<IPerMapConfiguration>` directly. Hence, the `IConfiguration` that instantiates those can be merged straight with the `DriverConfiguration`.

I think the confusion is mine, because we don't currently expose any way for the users of IMRU to directly add things to the Driver configuration, right?